tiledb/sm/array/array.cc

```diff
@@ -98,6 +98,11 @@ shared_ptr<const ArraySchema> Array::array_schema_latest_ptr() const {
   return array_schema_latest_;
 }
 
+void Array::set_array_schemas_all(
+    std::unordered_map<std::string, shared_ptr<ArraySchema>>& all_schemas) {
+  array_schemas_all_ = all_schemas;
```

Comment:
Thanks for catching this and providing a fix. My only suggestion is to ensure there's no issues with the rtree domain. When I first added this API [here](https://github.com/TileDB-Inc/TileDB/pull/2844/files#diff-74ce8c89d5884c4961474244d6263b78237ca72ec9e61c9a8268b4fab73b0b53R111), the rtree domain had to be updated when the fragment metadata was. Just a note; it may not be relevant anymore but I wanted to make sure it was mentioned.

Reply:
Thanks for letting me know about this, I didn't notice. However, I am not sure I understand the issue, is there is a way to see if I need to do that, a test that will fail otherwise or so?